app/views/shared/_header.html.slim

```diff
@@ -1,7 +1,7 @@
 header.row
-  .col-xs-6
-    .page-header
-      h2 SUSE&reg; Container as a Service Platform
-  .col-xs-6
-    - if user_signed_in?
-     = link_to('Logout', destroy_user_session_path, class: "btn btn-default pull-right", :method => :delete)
+  .col-xs-12
+    .page-header.col-xs-6
+        h2 SUSE&reg; Container as a Service Platform
+    .col-xs-6
+      - if user_signed_in?
+      = link_to('Logout', destroy_user_session_path, class: "btn btn-default pull-right", :method => :delete)
```

Comment:
Do you have any example working on another web application that handles users? I'm only seeing Get and Posts. (And Get is being highly discouraged because of browsers pre-fetching GET pages).

Stackoverflow users seem to prefer POST
http://stackoverflow.com/questions/5868786/what-method-should-i-use-for-a-login-authentication-request
http://softwareengineering.stackexchange.com/questions/196871/what-http-verb-should-the-route-to-log-out-of-your-web-app-be